### PR TITLE
Fix inaccurate item name in README

### PR DESCRIPTION
**Pull request by osawata36**
### Why:

This PR intends to fix an inaccurate item name.

### What's being changed:

Step4 in README.
Description of the item name on the screen to be checked in step 4 .

### Check off the following:

- [x] For workflow changes, I have verified the Actions workflows function as expected.
- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by rashad-masud** [comment]
dd

**Comment by rashad-masud** [comment]
dddd

**Comment by rashad-masud** [comment]
fff

File: README.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ Protected branches ensure that collaborators on your repository cannot make irre
 1. Go to **Branches** settings. You can navigate to that page manually by clicking on the right-most tab in the top of the repository called **Settings** and then clicking on **Branches**.
 1. Click on **Add rule** under "Branch protection rules".
 1. Type `main` in **Branch name pattern**.
-1. Check **Require pull request reviews before merging**.
+1. Check **Require a pull request before merging**.
```
Comment (sinsukehlab) [inline README.md:198]: Cf. #14
```suggestion
1. Check **Require a pull request before merging**.
1. Uncheck **Require approvals**.
```

Comment (sinsukehlab) [reply README.md:198]: Also, I wonder if l. 200 is necessary.

**Comment by heiskr** [comment]
I'll go ahead and merge this as is, I've internally requested an instructional designer to review this course given the comments on it.

**Pull request merged**